libs/shared/contentful/src/lib/queries/eligibility-content-by-offering/util.spec.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  EligibilityContentByOfferingQueryFactory,
+  EligibilityContentByOfferingResult,
+  EligibilityContentOfferingResultFactory,
+  EligibilityContentByOfferingResultUtil,
+} from '.';
+
+describe('EligibilityByOfferingResultUtil', () => {
+  it('should create a util from response', () => {
+    const result = EligibilityContentByOfferingQueryFactory();
+    const util = new EligibilityContentByOfferingResultUtil(
+      result as EligibilityContentByOfferingResult
+    );
+    expect(util).toBeDefined();
+    expect(util.getOffering()).toBeDefined();
+    expect(util.offeringCollection.items.length).toBe(1);
```

Comment:
Not sure if we have a strong preference between `expect(xyz.length).toBe(n)` vs `expect(xyz).toHaveLength(n)`:

```sh
git grep -Pin "\.toHaveLength\(" -- "libs/**/*.spec.*" | cat

libs/payments/capability/src/lib/capability.manager.spec.ts:78:      expect(result[0].capabilities).toHaveLength(6);
libs/shared/contentful/src/lib/contentful.manager.spec.ts:121:      expect(result.purchaseCollection.items).toHaveLength(0);
libs/shared/contentful/src/lib/contentful.manager.spec.ts:140:      ).toHaveLength(1);
libs/shared/contentful/src/lib/contentful.manager.spec.ts:220:      expect(result.serviceCollection.items).toHaveLength(0);
libs/shared/contentful/src/lib/contentful.manager.spec.ts:228:      expect(result.serviceCollection.items).toHaveLength(1);
libs/shared/contentful/src/lib/contentful.manager.spec.ts:244:      expect(result.purchaseCollection.items).toHaveLength(0);
```

```sh
git grep -Pin "\.length\)\.toBe\(" -- "libs/**/*.spec.*" | cat

libs/payments/capability/src/lib/capability.manager.spec.ts:50:      expect(result.length).toBe(0);
libs/payments/capability/src/lib/capability.manager.spec.ts:71:      expect(result.length).toBe(1);
libs/payments/capability/src/lib/capability.manager.spec.ts:116:      expect(Object.keys(result).length).toBe(0);
libs/payments/capability/src/lib/capability.manager.spec.ts:129:      expect(Object.keys(result).length).toBe(0);
libs/payments/capability/src/lib/capability.manager.spec.ts:149:      expect(Object.keys(result).length).toBe(0);
libs/payments/capability/src/lib/capability.manager.spec.ts:193:      expect(Object.keys(result).length).toBe(2);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:43:      expect(result.length).toBe(0);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:58:      expect(result.length).toBe(1);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:98:      expect(result.length).toBe(1);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:134:      expect(result.length).toBe(1);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:154:      expect(result.length).toBe(1);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:194:      expect(result.length).toBe(1);
libs/payments/eligibility/src/lib/eligibility.manager.spec.ts:252:      expect(result.length).toBe(2);
libs/payments/legacy/src/lib/plan-mapper.util.spec.ts:40:      expect(defaultMapper.errorFields.length).toBe(0);
libs/payments/legacy/src/lib/plan-mapper.util.spec.ts:52:      expect(defaultMapper.errorFields.length).toBe(1);
libs/payments/legacy/src/lib/plan-mapper.util.spec.ts:65:      expect(defaultMapper.errorFields.length).toBe(0);
libs/payments/legacy/src/lib/plan-mapper.util.spec.ts:77:      expect(defaultMapper.errorFields.length).toBe(0);
libs/payments/legacy/src/lib/plan-mapper.util.spec.ts:172:      expect(result.errorFields.length).toBe(1);
libs/payments/legacy/src/lib/stripe-mapper.service.spec.ts:173:      expect(nonMatchingPlans.length).toBe(0);
libs/payments/legacy/src/lib/stripe-mapper.service.spec.ts:212:      expect(nonMatchingPlans.length).toBe(0);
libs/shared/contentful/src/lib/queries/eligibility-content-by-offering/util.spec.ts:20:    expect(util.offeringCollection.items.length).toBe(1);
libs/shared/contentful/src/lib/queries/eligibility-content-by-plan-ids/util.spec.ts:25:    expect(util.purchaseCollection.items.length).toBe(1);
libs/shared/contentful/src/lib/queries/purchase-with-details-offering-content/util.spec.ts:31:    expect(util.purchaseCollection.items.length).toBe(2);
libs/shared/contentful/src/lib/queries/services-with-capabilities/util.spec.ts:18:    expect(util.serviceCollection.items.length).toBe(1);
```

Reply:
I prefer `expect(xyz).toHaveLength(n)`, but I think updating all of these files would be better in a follow-up PR.